modules/role/manifests/st2.pp

```diff
@@ -22,4 +20,12 @@
       include ::profile::hubot
     }
   }
+
+  if $_enterprise_token {
+    include ::profile::st2flow
+    include ::profile::st2rbac
+    include ::profile::enterprise_auth_backend_ldap
+  } else {
+    include ::profile::auth_backend_pam
```

Comment:
Well, LDAP should be an option for enterprise, but not required.

Only including LDAP and not pam in enterprise version also makes testing a lot harder since you now need LDAP server (@jfryman please correct me if I'm missing something).

Actually, this wouldn't work anyway - we need pam for the root_cli and admin user to work correctly, something, I believe we haven't accounted for yet so LDAP wouldn't work out of the box even if you had LDAP server available anyway...